docs/university.md

```diff
@@ -2506,4 +2506,4 @@ type 列表：
 
 ### 数据科学与计算机学院动态
 
-<Route author="Neutrino3316 MegrezZhu" example="/sysu/sdcs" path="/sysu/sdcs" />
+<Route author="Neutrino3316 MegrezZhu nczitzk" example="/sysu/cse" path="/sysu/cse/:first?/:second?/:third?" :paramsDesc="['招聘类型']"/>
```

Comment:
Wrong path and paramsDesc.
`first`, `second`, `third` are not used in the code.